aiven/resource_vpc_peering_connection.go

```diff
@@ -132,41 +133,57 @@ func resourceVPCPeeringConnectionCreate(ctx context.Context, d *schema.ResourceD
 		peerResourceGroup = v.(string)
 	}
 
-	pc, err = client.VPCPeeringConnections.Create(
+	if _, err = client.VPCPeeringConnections.Create(
 		projectName,
 		vpcID,
 		aiven.CreateVPCPeeringConnectionRequest{
-			PeerCloudAccount:     d.Get("peer_cloud_account").(string),
-			PeerVPC:              d.Get("peer_vpc").(string),
+			PeerCloudAccount:     peerCloudAccount,
+			PeerVPC:              peerVPC,
 			PeerRegion:           region,
 			UserPeerNetworkCIDRs: cidrs,
 			PeerAzureAppId:       peerAzureAppID,
 			PeerAzureTenantId:    peerAzureTenantID,
 			PeerResourceGroup:    peerResourceGroup,
 		},
-	)
-	if err != nil {
-		return diag.Errorf("Error creating VPC peering connection: %s", err)
+	); err != nil && !aiven.IsAlreadyExists(err) {
```

Comment:
`!aiven.IsAlreadyExist` should not be here. When we attempt to create a resource, an expectation is that it does not exist upfront. The reason is that preexisting resources might have a different configuration from the one we want to create, but both might have the same ID.

Reply:
done